src/components/widgets/Clock.js

```diff
@@ -65,4 +61,21 @@ class Clock extends Component {
   }
 }
 
-export default Clock
+const mapStateToProps = (state) => {
```

Comment:
Let's omit this, and just supply the props directly to the component, instead of mapping (since this will not be the structure of the store as we continue to build this out, I don't want to lock ourselves into `mapStateToProps` here just yet):
```jsx
  <Clock {...this.state} />
```